src/crimson/os/alienstore/alien_store.cc

```diff
@@ -335,8 +335,14 @@ AlienStore::get_attrs(CollectionRef ch,
   return seastar::do_with(attrs_t{}, [=] (auto &aset) {
     return tp->submit(ch->get_cid().hash_to_shard(tp->size()), [=, &aset] {
       auto c = static_cast<AlienCollection*>(ch.get());
-      return store->getattrs(c->collection, oid,
-		             reinterpret_cast<map<string,bufferptr>&>(aset));
+      std::map<std::string, ceph::bufferptr> blueaset;
+      const auto r = store->getattrs(c->collection, oid, blueaset);
```

Comment:
i'd suggest change the interface of ObjectStore, so that it returns a `map<...,less<>>`. working on a PR on the classic end.